language/German/strings.po

```diff
@@ -1225,40 +1208,38 @@ msgid ""
 "\n"
 "If this is your first time using %2$s, please see the config.txt file in the "
 "docs directory for information on configuring %2$s."
-msgstr ""
+msgstr "Keine Maschinen gefunden. Bitte überprüfen Sie den Pfad für die ROMs in Ini-Datei %1$s.ini.\n\nWenn dies Ihre erste Verwendung von  %2$s ist, konsultieren Sie bitte config.txt im docs Verzeichnis zur Konfiguration von %2$s."
 
```

Comment:
Mein Vorschlag: "Keine Maschinen gefunden. Bitte überprüfen Sie den in der ini-Datei %1$s.ini angegeben Pfad zu den ROM-Dateien. \n\n Wenn Sie %2$s zum ersten Mal verwenden, konsultieren Sie bitte config.txt im Verzeichnis "docs" zur Konfiguration von %2$s."

Ist aber (wie so oft bei Übersetzungen) eine Geschmacksfrage.